cmc-domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -27,7 +27,7 @@
     public static final LocalDateTime NOT_RESPONDED = null;
     public static final String SUBMITTER_EMAIL = "claimant@mail.com";
     public static final String DEFENDANT_EMAIL = SampleTheirDetails.DEFENDANT_EMAIL;
-
+    public static final String SEALED_CLAIM_DOCUMENT_MANAGEMENT_SELF_URL = "/self_url";
```

Comment:
could you add the new line back between the public and private fields please?

Reply:
added